pkg/policy/selectorcache.go

```diff
@@ -256,6 +284,15 @@ func (s *selectorManager) removeUser(user CachedSelectionUser) (last bool) {
 	return len(s.users) == 0
 }
 
+func (f *fqdnSelector) removeUser(user CachedSelectionUser) (last bool) {
+	delete(f.users, user)
+	removed := len(f.users) == 0
+	if removed {
+		f.NotifyRemoved()
```

Comment:
inlining `NotifyRemoved()`here would show the trouble I had with the name of `f.user`